commands/music/play.js

```diff
@@ -51,82 +51,86 @@ module.exports = class PlayCommand extends Command {
       { name: '3', urls: [ [Object], [Object] ] }
      ]
     */
-
-    if (db.get(message.member.id) !== null) {
-      const userPlaylists = db.get(message.member.id).savedPlaylists;
-      let found = false;
-      let location;
-      for (let i = 0; i < userPlaylists.length; i++) {
-        if (userPlaylists[i].name == query) {
-          found = true;
-          location = i;
-          break;
+    if (
```

Comment:
Why did you add this 'if' statement here?

Reply:
only lines 205 - 208 were Changed

Rest of play.js was formatted Via the settings that were set inside the `.prettierrc` file